src/app/components/job-list-item/job-list-item.component.scss

```diff
@@ -1,33 +1,34 @@
 @import '../../styles/colors';
 
 .adress-field {
-  padding-top: 2px;
-  padding-bottom: 2px;
   font-size: 14px;
+  padding-bottom: 2px;
+  padding-top: 2px;
 }
 
 .job-list-container {
+  border-bottom: 1px solid $color-gray-50;
```

Comment:
Properties should be ordered max-width, padding, margin, color, cursor, border-bottom

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)